Java/Core/build.gradle

```diff
@@ -59,14 +59,19 @@ android {
         // Default: required
         release {
             minifyEnabled false
-            //proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+
+            // Ensure that the -DNDEBUG compiler flag IS NOT set thereby ensuring that `assert(c)``
+            // evaluates `c` (for side-effects - but we've removed side-effects anyhow)
+            debuggable true
```

Comment:
This doesn't seem right. Will the app debuggable override this?

Reply:
Me commenting on build.gradle would be a guess, at best (event if GIT blame shows me).